doc/configuration.md

```diff
@@ -50,6 +50,14 @@ The Ignition configuration is a JSON document conforming to the following specif
       * **_id_** (integer): the user ID of the owner.
     * **_group_** (object): specifies the group of the owner.
       * **_id_** (integer): the group ID of the owner.
+  * **_directories_** (list of objects): the list of directories, rooted in this particular filesystem, to be created.
```

Comment:
The one it lists. 
I used the files section as a base for this. Admittedly both seem a little unclear; it looks like it was meant for the v1 spec. Change both to omit "rooted in this particular filesystem" and make the filesystem description include something along those lines?